tests/test_metadata/test_consolidated.py

```diff
@@ -503,7 +504,7 @@ async def test_consolidated_metadata_backwards_compatibility(
     async def test_consolidated_metadata_v2(self):
         store = zarr.storage.MemoryStore()
         g = await AsyncGroup.from_store(store, attributes={"key": "root"}, zarr_format=2)
-        dtype = "uint8"
+        dtype = parse_data_type("uint8", zarr_format=2)
```

Comment:
Is it not possible to just pass `"uint8"` any more?

Reply:
it is possible, i'm just not doing that in this test